config/development.json

```diff
@@ -46,5 +46,6 @@
 		"SECURE": false,
 		"EXPIRES_SECONDS": 2592000000
 	},
-	"HYDRA_URI": "http://localhost:9001"
+	"HYDRA_URI": "http://localhost:9001",
+	"FEATURE_COURSE_EXPORT_ENABLED": true
```

Comment:
For better developer experience we should leave this one, because otherwise we have to restart the auto deployment process every time we work on this feature. Because the environment variables will be only set during initialization and not for every commit.

Reply:
Agree